libtk205/include/libtk205.h

```diff
@@ -0,0 +1,38 @@
+#ifndef LIBTK205_H_
+#define LIBTK205_H_
+
+#include <vector>
+#include <memory>
+#include "ASHRAE205.h"
+#include "RS0001.h"
+#include "RS0002.h"
+#include "RS0003.h"
+#include "RS0004.h"
+#include "RS0005.h"
+#include "RS0006.h"
+#include "RS0007.h"
+#include "nlohmann/json.hpp"
+
+namespace libtk205_NS {
```

Comment:
Can we eliminate this namespace level in favor of `ASHRAE205_NS`?

Reply:
Makes sense. Will do.